contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -97,29 +107,29 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
   uint256 internal constant REPORT_METADATA_LENGTH = 105;
   uint256 internal constant SIGNATURE_LENGTH = 65;
 
-  function setConfig(uint32 donId, uint8 f, address[] calldata signers) external onlyOwner {
+  // TODO: separate call to set minConfigVersion? seems odd to call as part of setConfig since you'd usually bump it to latest value post-migration
```

Comment:
^

Reply:
Agree with a separate function. I'd call it `removeConfig` though and use it to set the `f` value to 0.